app/routes/lending/LendableObjectsAdmin.css

```diff
@@ -0,0 +1,6 @@
+.heading {
+  font-size: var(--font-size-lg);
+  border-bottom: 1px solid var(--border-gray);
```

Comment:
Not a fan of borders under headers since we don't really do that anywhere else on the webapp. Also, why override the font-size of `h2`? Please be consistent with other designs

Reply:
We have borders like this under headings all over the webapp... in joblistings, events, company interest, meetings, etc. The override of h2 is consistent with how headings is styled in eventlists, but I see now that headings are styled differently all over the webapp :face_with_spiral_eyes:

Reply:
If you prefer it any other way, I don't mind changing it though

Reply:
There are no h1 or h2 headers with border bottoms except for on events (but that's sort of an anomaly). But yes there are some smaller headers with border bottoms still living in the webapp - not a fan of those either.

Reply:
Okay sure, I don't really have a strong opinion on how these headings should look, so I'll make it however you want it:) But since we aren't consistent with it anywhere on the webapp, "be consistent with other designs" isn't gonna cut it. If you just tell me exactly how you want it, I'll fix it!

Reply:
I'm sorry, I was a bit crass yesterday, please do tell me how you would like it and I'll fix it, I really do appreciate the feedback<3